### DISCUSSION TITLE
[FIXED] Photos stopped uploading = shows blanks

### DISCUSSION THREAD

**Discussion by flokerama** [immich-app/immich#3764] [Q&A]
Hi. I've been uploading my photos today and after a while Immich stopped working. My uploads now show the broken/zigzag image. Sometimes, if I delete and retry, then some photos work but others won't. There's no pattern to it. Sometimes photo X will work. Then if I delete and try again it shows as broken. I haven't touched my config files at all. Am using defaults. Have tried docker-compose stop / start / down / up -d to get it going again. But no change. Please let me know how I can help with fixing this. Thanks. 

Edit: Am running 1,73 on NixOS.

**Comment by alextran1502** [comment]
Hello, can you verify the description of stop working?

Are the jobs in Administration > Jobs still running?

What are the logs/status of the server and microservices container?

**Reply by flokerama** [reply]
Hi. It says no jobs active or waiting. How can I check the logs (sorry for noob queston)?

**Reply by flokerama** [reply]
Hi. Checked the logs. The microserver logs says there is no space on the disc, even though there is tonnes (about 400GB) of space free.

**Reply by bo0tzz** [reply]
That would suggest your library folder is not properly mounted. Are files showing up where you would expect them to?

**Comment by flokerama** [comment]
I think so. But I was having some issues with it so I've nuked the drive and am setting immich up again on a separate one. So hopefully it will work this time.

**Comment by flokerama** [comment]
All fixed with a new drive.
